test/integration/testdata/test-buildcli.json

```diff
@@ -55,7 +55,7 @@
         "output": {
           "to": {
             "kind": "ImageStreamTag",
-            "name": "origin-ruby-sample:latest"
+            "name": "ruby-20-centos7-buildcli:latest"
```

Comment:
The original version of the file contained two identical BuildConfigs, `ruby-sample-build-validtag` and `ruby-sample-build-invalidtag`. There's also ImageStreamTag `ruby-20-centos7-buildcli` which is not referenced anywhere.

In order to test binary build with a BuildConfig it has to have correct output specification, otherwise the build pod is not created and the binary input is not sent. Thus this change.

I can create separate fixture for this test if the result is less confusing.